hazelcast/src/main/java/com/hazelcast/internal/cluster/impl/ClusterHeartbeatManager.java

```diff
@@ -103,9 +110,34 @@
         heartbeatIntervalMillis = getHeartbeatInterval(hazelcastProperties);
         pingIntervalMillis = heartbeatIntervalMillis * HEART_BEAT_INTERVAL_FACTOR;
```

Comment:
Can you rename `pingIntervalMillis ` to something more explicit to not to confuse with `icmpIntervalMillis `? It's wrongly named, it's not a ping interval but a threshold to ping when there are not heartbeats in some period...